dts/msc-sm2-imx6x.dtsi

```diff
@@ -446,6 +446,48 @@
 			regulator-max-microvolt = <3300000>;
 			regulator-always-on;
 		};
+
```

Comment:
Since i know this is also work in progress and waiting for msc support on why the min level setting do not work, i suggest to put this in its own pull request.